Sso/config/services.php

```diff
@@ -35,12 +35,19 @@
         'secret' => env('STRIPE_SECRET'),
     ],
 
-    'google' => [
-        'client_id' => env('GOOGLE_CLIENT_ID', 'your-github-app-id'),
-        'client_secret' => env('GOOGLE_CLIENT_SECRET', 'your-github-app-secret'),
-        'redirect' => env('GOOGLE_CALLBACK_URL', 'http://your-callback-url'),
+    'auth' => [
```

Comment:
Socialite library depends on `services.google` configuration, I'll have to revert this back.